workshop.wwwapi/Endpoints/SurgeryEndpoint.cs

```diff
@@ -11,19 +49,44 @@ public static void ConfigurePatientEndpoint(this WebApplication app)
             var surgeryGroup = app.MapGroup("surgery");
 
             surgeryGroup.MapGet("/patients", GetPatients);
+            surgeryGroup.MapGet("/patient", GetPatient);
```

Comment:
should be `/patients/{id}` -> you need to specify the ID in the URL otherwise how do you know which patient to get?